moveit_core/package.xml

```diff
@@ -53,6 +53,7 @@
   <depend>tf2_geometry_msgs</depend>
   <depend>trajectory_msgs</depend>
   <depend>visualization_msgs</depend>
+  <depend>ruckig</depend>
```

Comment:
I had assumed this would be an optional plugin, but it appears it will be a required dependency. Given the experimental state of the results you presented in this PR, is there a way to reduce the impact of this?

Reply:
Can this be added instead to [planning_request_adapter_plugins](https://github.com/ros-planning/moveit2/tree/main/moveit_ros/planning/planning_request_adapter_plugins/src)?

Reply:
All trajectory processing algorithms are implemented in moveit_core, only the plugin wrappers are provided by the referenced package. I think it's ok to add this dependency here, especially since it has been released in the meantime.